src/devices/Mcp25xxx/Mcp25xxx.cs

```diff
@@ -241,6 +248,74 @@ public void Reset()
             _spiDevice.WriteByte((byte)InstructionFormat.Reset);
         }
 
+        /// <summary>
+        /// If RXB0 contains a valid message and another valid message is received,
+        /// an overflow error will not occur and the new message will be moved into RXB1
+        /// </summary>
+        public void EnableRollover()
+        {
+            WriteByte(
+                new RxB0Ctrl(
+                    false,
+                    true,
+                    false,
+                    OperatingMode.TurnsMaskFiltersOff));
+        }
+
+        /// <summary>
+        /// Set CAN Bitrate
+        /// </summary>
+        public void SetBitrate(Bitrates cnf1, Bitrates cnf2, Bitrates cnf3)
+        {
+            WriteByte(Address.Cnf1, (byte)cnf1);
+            WriteByte(Address.Cnf2, (byte)cnf2);
+            WriteByte(Address.Cnf3, (byte)cnf3);
+        }
+
+        /// <summary>
+        /// Set mode
+        /// </summary>
+        public void SetMode(OperationMode operationMode)
+        {
+            WriteByte(
+                new CanCtrl(
+                    CanCtrl.PinPrescaler.ClockDivideBy8,
+                    true,
+                    false,
+                    false,
+                    operationMode));
+        }
+
+        /// <summary>
+        /// Read messages from all buffers
+        /// </summary>
+        /// <param name="byteCount">Number of bytes to read.  This must be one or more to read.</param>
+        /// <returns>Array of messages</returns>
+        /// <exception cref="ArgumentOutOfRangeException"></exception>
+        public byte[][] ReadMessages(int byteCount)
```

Comment:
`byte[][]` as return type here is a bit ugly to use for the caller. Can we use `List<byte[]>` instead (as it is a list of messages, not a matrix)? Or maybe even something more elegant?

Also, I'm confused about why we need to specify `byteCount` here. As far as I know, the length of each CAN-Bus message is encoded in the data stream. How would an user know how many bytes to read before looking at the messages?